ara/infra/notification.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import HTTPException
+from sqlalchemy.orm import Session
+from ara.domain.notification import Notification
+from ara.domain.user import User
+from ara.domain.exceptions import EntityDoesNotExist
+from typing import List
+
+class NotificationRepository:
+    def __init__(self, db: Session):
+        self.db = db
+
+    def get(self, notification_id: int) -> Notification:
```

Comment:
```suggestion
    def get_by_id(self, notification_id: int) -> Notification:
```
함수이름은 파라미터를 보지 않아도 딱봐도 알 수 있게 작성하는게 좋아요 👍